quartz/plugins/transformers/frontmatter.ts

```diff
@@ -43,14 +45,14 @@ function coerceToArray(input: string | string[]): string[] | undefined {
     .map((tag: string | number) => tag.toString())
 }
 
-export function getAliasSlugs(aliases: string[], argv: Argv, file: VFile): FullSlug[] {
-  const dir = path.posix.relative(argv.directory, path.dirname(file.data.filePath!))
+export function getAliasSlugs(aliases: string[], _argv: Argv, file: VFile): FullSlug[] {
   const slugs: FullSlug[] = aliases.map(
-    (alias) => path.posix.join(dir, slugifyFilePath(alias as FilePath)) as FullSlug,
+    (alias) => (alias.startsWith("/") ? alias : `/${alias}`) as FullSlug,
```

Comment:
hm i think this is a breaking change that might also be incorrect, https://help.obsidian.md/aliases i believe implies that aliases just rename the last segment of the note (and its also how I use it within my vault)

Reply:
Yeah, you're right.

It also seems that permalinks work differently than we have implemented (still checking this at the moment.) According to the [Obsidian docs on permalinks](https://help.obsidian.md/publish/permalinks), it seems that the permalink should be the note URL and the note's actual path should be the redirect. I think we have this backwards at the moment.

Reply:
hmm, for permalink I think it means we will need to convert all wikilinks to this target as well (We can probably also do it in transclude under renderPage fwiw)

Reply:
I have path rewriting built into Quartz Syncer. I'll add an option there to either match Quartz or Obsidian Publish permalinks. Proper rewriting is probably v5 territory.

Reply:
> hm i think this is a breaking change that might also be incorrect, https://help.obsidian.md/aliases i believe implies that aliases just rename the last segment of the note (and its also how I use it within my vault)

Okay, I just found out you're half right. That is how aliases work in Obsidian *if* you only use a name (so no path). However, in Obsidian Publish, permalinks **do** use the aliases as full path relative to site root.

https://help.obsidian.md/publish/permalinks#Redirect+old+notes